### ISSUE TITLE
Official plugin registry

### ISSUE THREAD

**Issue by Justineo** [less/less-docs#283]
Hi all,

Since Less now supports plugins, I think it would be sweet if we have an "official (recommended) plugin registry/list". (We kind of have one in current docs but I think we may need a mechanism of publish/register/submit plugins from the community.) It can prevent similar functionalities from being implemented several times and save developers lots of efforts when looking for suitable plugins.

**Comment by matthew-dean** [comment]
Like a package manager? What about using NPM?

**Comment by lukeapage** [comment]
I think its overkill. The majority of the plugins are tiny post-process plugins. as Matthew says, you can search on npm and a PR to our docs isn't difficult. If we start getting 100's of plugins we can revisit.

**Comment by matthew-dean** [comment]
Agreed. The docs can get pull requests from people if others want to add to the list. If we get a longer list it can break off into its own section first, and then if the list gets too long too manage, decide then if we want to do something different.

Having said that, I am interested in consuming a list of plugins in a data (JSON) format. @lukeapage are you tagging all plugins with "less plugins" as keywords in NPM? If so, maybe we could recommend that in docs, or some common way of tagging so it could be consumed with a search to: https://www.npmjs.com/browse/keyword/less%20plugins. Then it would be a kind of "unofficial-official" list in a package manager.

**Comment by bassjobsen** [comment]
I recently post some question here https://github.com/less/less-meta/issues/5. Not the right place or not interesting. In summary i wonder how to distribute client side plugins?

**Comment by Justineo** [comment]
I noticed jQuery recently switched to NPM to host its plugins. This NPM blog post recommended to add `jquery-plugin` and `ecosystem:jquery` as keywords (they said "make sure to do it"). So what if we do the same way by adding `less-plugin` & `ecosystem:less` for Less plugins?

**Comment by bassjobsen** [comment]
+1 i will add the keywords from now

**Comment by seven-phases-max** [comment]
To note increase the entropy of the universe I'll be using `less-plugin` keyword only.

**Comment by Justineo** [comment]
Do things according to specific rules help slow down entropy increment.
